docs/docusaurus.config.js

```diff
@@ -1,108 +1,147 @@
 const path = require('path');
-const remarkSources = require('remark-sources');
-const remarkCollapse = require('remark-collapse');
-const { remarkProgramOutput } = require("./plugins/program_output");
+const { remarkProgramOutput } = require('./plugins/program_output');
+const { rehypePlugins, remarkPlugins } = require('@rasahq/docusaurus-theme-tabula');
```

Comment:
i'd rename them `rehypeThemePlugins` and `remarkThemePlugins` respectively, as it's more explicit when reading the config file

Reply:
👍 can do